Lib/enum.py

```diff
@@ -614,7 +614,7 @@ def __dir__(self):
                 for cls in self.__class__.mro()
                 for m in cls.__dict__
                 if m[0] != '_' and m not in self._member_map_
-                ]
+                ] + [m for m in self.__dict__ if m[0] != '_']
```

Comment:
Personally I'd prefer `if not m.startswith("_")` because it's more Pythonic and resilient in case someone smuggles an empty string in.
But, perhaps, `m[0]` is more performant?

Reply:
Performance isn't really an issue for a function like `dir()` (except in extreme cases, of course).  In this case we'll stick with `m[0]` to match what's already being used.

Reply:
It seems that `m[0]` is more performant and I also wanted to keep the consistency with the previous comparison `m[0] != '_' `

I don't know why it more performant though, I've found the following infos about what is performed:
- `m[0]` uses `COMPARE_OP` which seems to use opcode prediction, paired with `POP_JUMP_IF_FALSE` and `POP_JUMP_IF_TRUE`
- `startswith` implies a call of the builtin function

Thanks for asking, I've learned more on the opcode dispatch while investigating!